src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let attemptsCount = 0;
+
+  return function getSum(a) {
+    attemptsCount++;
+
+    return function second(b) {
+      if (attemptsCount <= 3 || attemptsCount % 2 === 1) {
+        return a + b;
+      }
```

Comment:
you can handle error first, and when there is no erorr - we can just return our sum